docs/ftldns/telnet-api.md

```diff
@@ -1,180 +1,216 @@
+
+#### `>querytypes` {data-toc-label='querytypes'}
+
+Get collected query types percentage
+
+```text
+A (IPv4): 53.45
+AAAA (IPv6): 45.32
+ANY: 0.00
+SRV: 0.64
+SOA: 0.05
+PTR: 0.54
+TXT: 0.00
+```
+
+#### `>getallqueries` {data-toc-label='getallqueries'}
+
+Get all queries that FTL has in memory
+
+ ```text
+1525554586 A fonts.googleapis.com 192.168.2.100 3 0 4 6
+1525554586 AAAA fonts.googleapis.com 192.168.2.100 3 0 4 5
+1525554586 A www.mkdocs.org 192.168.2.100 3 0 4 7
+1525554586 AAAA www.mkdocs.org 192.168.2.100 2 0 3 21
+1525554586 A squidfunk.github.io 192.168.2.100 2 0 3 20
+1525554586 A pi-hole.net 192.168.2.100 3 0 4 5
+1525554586 AAAA squidfunk.github.io 192.168.2.100 3 0 1 6
+1525554586 AAAA pi-hole.net 192.168.2.100 2 0 1 18
+1525554586 A github.com 192.168.2.100 3 0 4 5
+1525554586 AAAA github.com 192.168.2.100 2 0 1 18
+```
+
+Variants: `>getallqueries (37)` show (up to) 37 latest entries, `>getallqueries-time 1483964295 1483964312` gets all queries that FTL has in its database in a limited time interval, `>getallqueries-time 1483964295 1483964312 (17)` show matches in the (up to) 17 latest entries, `>getallqueries-domain www.google.com` gets all queries that FTL has in its database for a specific domain name, `>getallqueries-client 2.3.4.5` : gets all queries that FTL has in its database for a specific client name *or* IP
+
+#### `>recentBlocked` {data-toc-label='recentBlocked'}
+
+Get most recently pi-holed domain name
+
+```text
+www.googleadservices.com
+```
+
+Variant: `>recentBlocked (4)` show the four most recent blocked domains
+
+#### `>clientID` {data-toc-label='clientID'}
+
+Get ID of currently connected client
+
+```text
+6
+```
+
+#### `>version` {data-toc-label='version'}
+
+Get version information of the currently running FTL instance
+
+```text
+version v1.6-3-g106498d-dirty
+tag v1.6
+branch master
+hash 106498d
+date 2017-03-26 13:10:43 +0200
+```
+
+#### `>dbstats` {data-toc-label='dbstats'}
+
+Get some statistics about `FTL`'s' long-term storage database (this request may take some time for processing in case of a large database file)
+
+```text
+queries in database: 2700304
+database filesize: 199.20 MB
+SQLite version: 3.23.1
+```
+
+#### `>domain pi-hole.net` {data-toc-label='domain'}
+
+Get detailed information about domain (if available)
+
+```text
+Domain "pi-hole.net", ID: 254
+Total: 179
+Blocked: 0
+Wildcard blocked: false
+```
+
+#### `>cacheinfo` {data-toc-label='cacheinfo'}
+
+Get DNS server cache size and usage information
+
+```text
+cache-size: 500000
+cache-live-freed: 0
+cache-inserted: 15529
+```
+
+#### `>dns-port` {data-toc-label='dns-port'}
+
+Get DNS port FTL is listening on
+
+```text
+53
+```
+
+Note that the port can also be `0` if someone decides to disable the DNS server part of Pi-hole
+
+#### `>maxlogage` {data-toc-label='maxlogage'}
+
+Get timespan of the statistics shown on the dashboard (in seconds)
+
+```text
+86400
+```
+
+#### `>gateway` {data-toc-label='gateway'}
```

Comment:
I saw you used `####`. This generates a `<h4>` tag.

I used `<h2>` in my test.

I'm not sure which size is better.
`<h3>` uses a bigger font-size and have a bigger top-margin. `<h2>` is even bigger.

What do you think? `h4`, `h3` or `h2`?